modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/GlobalSnapshotStateChannelAcceptanceManager.scala

```diff
@@ -37,10 +35,11 @@ trait GlobalSnapshotStateChannelAcceptanceManager[F[_]] {
 
 object GlobalSnapshotStateChannelAcceptanceManager {
   def make[F[_]: Async: KryoSerializer](
-    ordinalDelay: Option[PosLong],
-    stateChannelAllowanceLists: Option[Map[Address, NonEmptySet[PeerId]]]
+    stateChannelAllowanceLists: Option[Map[Address, NonEmptySet[PeerId]]],
+    pullDelay: NonNegLong = NonNegLong.MinValue,
+    purgeDelay: NonNegLong = NonNegLong(4L)
```

Comment:
Even if it's set to a minimum value the event will get processed once. With both pull and purge delay set to 0 events will get processed so it's a good setting for validators only, e.g. snapshot-streaming

Reply:
could you move that default value to some const?

Reply:
something like `defaultPurgeDelay`?

Reply:
can be. anything that prevents magic numbers in the code